src/main/kotlin/org/jetbrains/research/tasktracker/tracking/task/Task.kt

```diff
@@ -2,6 +2,7 @@ package org.jetbrains.research.tasktracker.tracking.task
 
 interface Task {
```

Comment:
Why task is an interface, not data class? (as far as I can see, it's functionality is only can store data)

Reply:
This wasn't written by me, but I assume that "Task" is an interface because the logic for obtaining these fields may become much more complex when we work with the server-side.

Reply:
Ok, let's leave it as is and return to this question when the server functionality will arrive